### PR TITLE
Paulcentea branch

### PR DESCRIPTION
**Pull request by CenteaPaul**
added paul centea tasks

### CODE REVIEW

File: paul-centea-dir/paulcentea.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+paul centea created this file during Lab 5
+second commit
```
Comment (ionut0) [inline paul-centea-dir/paulcentea.txt:2]: reviewed by Ionut Poplacean

**Review by ionut0 — CHANGES_REQUESTED** [review]

File: paul-centea-dir/paulcentea.txt (hunk 1 above)
Comment (CenteaPaul) [reply paul-centea-dir/paulcentea.txt:2]: done

**Review by ionut0 — APPROVED** [review]

**Review by FlorinCiobotea — CHANGES_REQUESTED** [review]
Add "reviewed by FlorinCiobotea"

**Review by FlorinCiobotea — APPROVED** [review]

File: paul-centea-dir/paulcentea.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+reviewed by ionutPoplacean
+reveiwed by FlorinCiobotea
```
Comment (MelittaPirjolea) [inline paul-centea-dir/paulcentea.txt:4]: add reviewed by MelittaPirjolea

**Review by MelittaPirjolea — CHANGES_REQUESTED** [review]

**Review by MelittaPirjolea — APPROVED** [review]

**Pull request merged**